controllers/suite_test.go

```diff
@@ -153,6 +155,75 @@ var _ = BeforeSuite(func(done Done) {
 	deployerCSV.Spec.InstallStrategy.StrategySpec.DeploymentSpecs = []opv1a1.StrategyDeploymentSpec{}
 	Expect(k8sClient.Create(ctx, deployerCSV)).ShouldNot(HaveOccurred())
 
+	// create a mock OCS CSV
+	ocsCSV := &opv1a1.ClusterServiceVersion{}
+	ocsCSV.Name = ocsOperatorName
+	ocsCSV.Namespace = testPrimaryNamespace
+	ocsCSV.Spec.InstallStrategy.StrategyName = "test-strategy"
+	ocsCSV.Spec.InstallStrategy.StrategySpec.DeploymentSpecs = []opv1a1.StrategyDeploymentSpec{
```

Comment:
Move the mock data into a separate func, as having it here just inflate the current flow without a reason
ocsCSV.Spec.InstallStrategy.StrategySpec.DeploymentSpecs = getMockOCSCSVDeploymentSpec()